### PR TITLE
Implemented AR DNI

### PR DESCRIPTION
**Pull request by Thenila**

### CODE REVIEW

File: NumberValidator/Validators/AR/DNI.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            try
+            {
+                Validate(dni);
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+
```
Comment (pradeepgururani) [inline NumberValidator/Validators/AR/DNI.cs]: remove extra line break.

File: NumberValidator/Validators/AR/DNI.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                if (dni.Length != 7  || dni.Length != 8)
+                { 
+                throw new InvalidLengthException();
+                }
+                if (!dni.IsDigits())
+            {
+                throw new InvalidFormatException();
+            }
+              
+
```
Comment (pradeepgururani) [inline NumberValidator/Validators/AR/DNI.cs]: remove extra line breaks.

File: NumberValidator/Validators/AR/DNI.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿
+using System;
+using System.Globalization;
+using NumberValidator.Helpers;
+using System.Text.RegularExpressions;
+using NumberValidator.Validators.PAN;
+
+namespace NumberValidator.Validators.AR
+{
+     public class DNI : IValidator
```
Comment (pradeepgururani) [inline NumberValidator/Validators/AR/DNI.cs]: code formatting seems to be broken in this file. Please format it.

File: NumberValidator/Validators/IN/PAN.cs [hunk 4]
```diff
@@ -4,6 +4,7 @@
 using NumberValidator.Validators.AD;
 using System.Text.RegularExpressions;
 
+
```
Comment (pradeepgururani) [inline NumberValidator/Validators/IN/PAN.cs]: this file just with a new line break should not have been part of your PR.

**Comment by pradeepgururani** [comment]
You have not retained the original copyright messages. those things have to be copied over and to be added to our implementation.

**Comment by Thenila** [comment]
> You have not retained the original copyright messages. those things have to be copied over and to be added to our implementation.

I added the copyright messages

**Review by pradeepgururani — APPROVED** [review]

**Pull request merged**